demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/Batch.java

```diff
@@ -0,0 +1,51 @@
+package com.novoda.downloadmanager.demo.extended.batches;
+
+import com.novoda.downloadmanager.lib.DownloadManager;
+
+public class Batch {
```

Comment:
How does it go together with @ouchadam 's statement https://github.com/novoda/download-manager/pull/59#discussion_r33864682 
Could we update the README? Different PR though.